tool/gravity/cli/update.go

```diff
@@ -105,7 +105,7 @@ func newUpdater(ctx context.Context, localEnv, updateEnv *localenv.LocalEnvironm
 		return nil, trace.Wrap(err)
 	}
 	req := init.updateDeployRequest(deployAgentsRequest{
-		clusterState: cluster.ClusterState,
+		clusterState: clusterStateFromPlan(*plan),
```

Comment:
I'm curious, why can't we keep using ClusterState from the cluster object here?

Reply:
I'll add a comment - it cannot be safely done since only the update operation plan list of servers contains the correct view of cluster servers w.r.t cluster roles.